src/common.tests/ConsoleOutputTests.cs

```diff
@@ -56,7 +56,7 @@ public void SingleItem()
             const string ExpectedTestRow = "name1          Name2  name3";
             const string ExpectedTestItemJson = @"[
               {
-                ""name1"": ""name1"",

```

Comment:
@Eneuman the format is changed due to STJ changes ? Trying to understand why this is changed.

Reply:
@hsubramanianaks Nice catch, STJ uses PascalCase formating as default and I had missed adding PropertyNamingPolicy when console logging json. This has now been added.

Originaly I change the test because there was no `JsonPropertyName` attribute on the Name1 property in `TestOutputData` so I tought the test was wrong.

Theese changes only affect logging json to the console window.